config/default/ontology.conf.php

```diff
@@ -3,13 +3,15 @@
 use core_kernel_persistence_smoothsql_SmoothModel as SmoothModel;
 use oat\generis\model\kernel\persistence\smoothsql\search\ComplexSearchService;
 
-return new \core_kernel_persistence_smoothsql_SmoothModel([
+return new SmoothModel([
```

Comment:
IMO it is better to not have the alias at all, since we can clearly see what class we are talking about.

```suggestion
return new core_kernel_persistence_smoothsql_SmoothModel([
```